### PR TITLE
react countdown without knob

### PR DESCRIPTION
**Pull request by SarahNox**

### CODE REVIEW

**Comment by sharnik** [comment]
6 301 files changed! 😱

Those files you should remove from the git repository:
- [ ] .DS_Store
- [ ] the whole build directory
- [ ] countdown-browserified.js is not used for anything anymore, right?
- [ ] countdown-react.html, neither?
- [ ] node_modules are auto-generated / downloaded from package.json, they shouldn't be in the git repository

This should clean up the PR a little bit ;)

**Comment by SarahNox** [comment]
ok, 4 of 5 are done

File: src/Countdown.js [hunk 1]
```diff
@@ -1,17 +1,13 @@
...
-  <script src="https://cdnjs.cloudflare.com/ajax/libs/jquery/2.2.2/jquery.min.js"></script>
-  <script src="https://cdnjs.cloudflare.com/ajax/libs/marked/0.3.5/marked.min.js"></script>
-</head>
-<body>
-  <div id="countdown"></div>
-  <script type="text/babel">
+
+var React = require('react');
+ 
+// module.exports = React.createClass({
```
Comment (sharnik) [inline src/Countdown.js]: This commented out stuff is not needed.

Comment (SarahNox) [inline src/Countdown.js]: done

File: .gitignore [hunk 2]
```diff
@@ -0,0 +1,3 @@
+.DS_STORE
```
Comment (sharnik) [inline .gitignore]: The name of the file is  .DS_Store (your filesystem is case-sensitive).

**Pull request merged**